### DISCUSSION TITLE
Logo Design Request: `Joyboy` project

### DISCUSSION THREAD

**Discussion by AbdelStark** [stark-rocket/.github#18] [Q&A]
## Logo Design Request: "Joyboy"

**Project Overview:**
I'm looking for a designer to create a minimalistic and modern logo for a project named "Joyboy." The logo should be inspired by the Luffy "Joyboy" mode, featuring only the face with a hand covering the eyes, mouth open in a loud laugh, and smokey white hairs. It should convey joy and freedom, maintaining a clean and elegant style.

**Requirements:**
- Face-only illustration: hand covering eyes, open laughing mouth, smokey white hairs.
- Style: Minimalistic, modern, clean, and elegant.
- Formats: High-resolution JPG, transparent PNG, and vector files (AI, EPS).

**Deliverables:**
- Final logo in specified formats.
- Option for revisions based on initial feedback.

Please include examples of similar work in your proposal. Looking forward to collaborating!

![JBOB](https://github.com/keep-starknet-strange/.github/assets/45264458/1e7d3e38-900d-4054-9aad-64befa5fafd0)

**Comment by lamkuro** [comment]
I'm interested

**Reply by AbdelStark** [reply]
Let's give it a try. Continuing the discussion in DMs on telegram.
